### PR TITLE
Enable columns names by default for CSV in issue #14007 solved

### LINKED ISSUE #14007: Enable columns names by default for CSV exports
The CSV Excel exporter has this enabled but the normal one doesn't.

### PR DESCRIPTION
**Pull request by rijaspk**
Enable columns names by default for CSV
Added a export  type check for CSV 
IF true added csv columns 
fixes #14007

Signed-off-by: Rijas Ahammed PK Rijaspk5195@gmail.com

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/classes/Plugins/Export/ExportCsv.php [hunk 1]
```diff
@@ -113,7 +114,9 @@ protected function setProperties()
     public function exportHeader()
     {
         global $what, $csv_terminated, $csv_separator, $csv_enclosed, $csv_escaped;
-
+        //Enable columns names by default for CSV
+        if($what == 'csv')
+            $GLOBALS['csv_columns'] = 'yes';
```
Comment (mauriciofauth) [inline libraries/classes/Plugins/Export/ExportCsv.php]: Hi, @rijaspk. Thanks for your contribution.
Before we can merge your pull request, we would like you to make the following changes.
The body of the `if` structure must be enclosed by braces and there must be a space between the `if` keyword and the parenthesis.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: libraries/classes/Plugins/Export/ExportCsv.php (hunk 1 above)
Comment (rijaspk) [reply libraries/classes/Plugins/Export/ExportCsv.php]: Hi, @mauriciofauth. Required changes are implemented.

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Comment by rijaspk** [comment]
Hi, @mauriciofauth Thanks for reveiwing and accepting my PR

**Pull request merged**